.github/workflows/node.js.yml

```diff
@@ -18,9 +18,10 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
+      uses: actions/setup-node@v2
       with:
         node-version: ${{ matrix.node-version }}
+        cache: 'npm'
```

Comment:
```suggestion
```

If you don't want to cache your npm dependencies, apply this suggestion to remove the line.